CHANGES.md

```diff
@@ -1,3 +1,38 @@
+## edge-19.8.2
+
+**Significant Update**
```

Comment:
I didn't really understand this last time and I definitely don't understand it this time... what does this signify? I think we should take it out.